projects/angular/src/modal/_properties.modal.scss

```
@@ -19,14 +24,27 @@
         #{color.blue(variables.$clr-color-neutral-1000)},
         0.2
       );
+      --clr-modal-content-box-shadow: 0
+        #{variables.$clr_baselineRem_1px}
+        #{variables.$clr_baselineRem_2px}
+        #{variables.$clr_baselineRem_2px}
+        var(--clr-modal-content-box-shadow-color);
       --clr-modal-backdrop-color: var(--clr-color-neutral-900);
       --clr-modal-backdrop-opacity: 0.85;
       --clr-modal-border-radius: var(--clr-global-borderradius);
+      --clr-modal-content-border-style: none;
+      --clr-modal-content-border-color: transparent;
+      --clr-modal-content-border-width: 0;
+      --clr-modal-content-border-radius: unset;
+      --clr-modal-content-padding: #{variables.$clr_baselineRem_1};
```

Shouldn't that be like original state?

```suggestion
      --clr-modal-content-padding: #{variables.$clr_baselineRem_0_5 0 variables.$clr_baselineRem_0_5 variables.$clr_baselineRem_1};
```

---

Same thing here, what you are referring is the small variant.